### ISSUE TITLE
Vulnerabilities: Broken link for weakness

### ISSUE THREAD

**Issue by k-c-p** [DependencyTrack/dependency-track#690]
In the Vulnerabilites view the link in the section "Weakness" is broken.

For instance for vulnerability "CVE-2019-17267 (NVD)" (to be found at `https://$deptrackhost/vulnerabilities/NVD/CVE-2019-17267`) the link target behind "CWE-20: Improper Input Validation" is
```
https://$deptrackhost/vulnerabilities/NVD/<a href="http://cwe.mitre.org/data/definitions/20
```

which will yield a HTTP 404 when clicked on.

It should be a proper link.

Does not affect NVD vulnerabilites only, but NPM ones as well.

### Environment:
  
  - Dependency-Track Version: 3.8.0
  - Distribution: WAR in Tomcat9
  - Database Server: PostgreSQL

**Comment by stevespringett** [comment]
Confirmed to be a defect. Thanks for reporting.

**Comment by stevespringett** [comment]
Closing. Fixed in v4.0 - to be released soon.
